### PR TITLE
Pontoon: document new access key setting for locales

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: src/tools/pontoon/adding_new_locale.md [hunk 1]
```diff
@@ -105,4 +105,14 @@ Population: 1863468
 
 Find the locale code in [CLDR territoryInfo.json](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) and multiply its `_populationPercent` with the territory `_population`. Repeat if multiple occurrences of locale code exist and sum products.
 
-At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should appear automatically under the new team's page — in this case at https://pontoon.mozilla.org/am/
+At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should appear automatically under the new team’s page — in this case at https://pontoon.mozilla.org/am/
+
+### Access key localization
```
Comment (flodolo) [inline src/tools/pontoon/adding_new_locale.md:110]: @mathjazz 
The checkbox and description currently disagree on how to spell access key. I personally suggest to use the one with a space.

**Comment by flodolo** [comment]
As for which locales to enable, based on [Firefox data](https://docs.google.com/spreadsheets/d/18kFY5QKIUxmUlLWv8t-TnhksdL58vn6WBbCMSwhhZFM/edit#gid=0): bn, bo, brx, gu-IN, hi-IN, ja, kn, ko, mr, my, ne-NP, pa-IN, si, skr, te, zh-CN, zh-TW.

I've removed `en-CA`, `en-GB`, `sco`, as they might need to change access key when the label uses a different spelling. Also removed a few with Latin script, like `ro` or `es-CL`, because they clearly have no idea what access keys are and how they should be localized :-(

File: src/tools/pontoon/adding_new_locale.md (hunk 1 above)
Comment (mathjazz) [reply src/tools/pontoon/adding_new_locale.md:110]: Opened a PR to fix the auto-generated spelling:
https://github.com/mozilla/pontoon/pull/2857

File: src/tools/pontoon/adding_new_locale.md [hunk 2]
```diff
@@ -105,4 +105,14 @@ Population: 1863468
...
+
+In software, access keys allow users to activate a menu command using their keyboard, and they’re displayed as an underlined character in the menu label. Some locales, especially those using a script different than Latin (Chinese, Japanese, etc.), don’t localize access keys, and use the same character as English. In this case, the access key will be appended to the label as an underlined character between parentheses.
+
+Pontoon can recognize an access key attribute in Fluent strings, and provide translators with a dynamic UI to select one of the characters available in the localized label.
+
+When the checkbox is selected (default), Pontoon will leave the access key field empty, and show the dynamic UI when translating. For pretranslation, it will use the first character from the pretranslated label.
+
+When the checkbox is deselected, Pontoon will automatically use the same access key as the source string, for both manual translation and pretranslation.
```
Comment (mathjazz) [inline src/tools/pontoon/adding_new_locale.md:118]: Note that this is not implemented yet:
https://github.com/mozilla/pontoon/issues/2855

**Review by mathjazz — APPROVED** [review]

File: src/tools/pontoon/adding_new_locale.md (hunk 2 above)
Comment (flodolo) [reply src/tools/pontoon/adding_new_locale.md:118]: Thanks, I decided to go ahead and update the document once 👍🏼

**Pull request merged**